package.json

```diff
@@ -54,6 +54,7 @@
     "@ngrx/effects": "6.1.2",
     "@ngrx/router-store": "6.1.2",
     "@ngrx/store": "6.1.2",
+    "@ngx-formly/bootstrap": "^5.5.7",
```

Comment:
We do not want any dependency on bootstrap (even formly's). We should recreate any wrappers or types using uswds patterns. But I don't think this ended up being used so it can be removed.